kani-driver/src/project.rs

```diff
@@ -49,15 +35,6 @@ pub struct Project {
     pub outdir: PathBuf,
     /// The collection of artifacts kept as part of this project.
     artifacts: Vec<Artifact>,
-    /// A flag that indicated whether all artifacts have been merged or not.
-    ///
-    /// This allow us to provide a consistent behavior for `--function`.
-    /// For this option, we still merge all the artifacts together, so the
-    /// `merged_artifacts` flag will be set to `true`.
-    /// When this flag is `true`, there should only be up to one artifact of any given type.
-    /// When this flag is `false`, there may be multiple artifacts for any given type. However,
-    /// only up to one artifact for each
-    pub merged_artifacts: bool,
```

Comment:
So `merged_artifacts` in a project would only be used through the `--function` flag?

Reply:
I need to stop the review, I'll continue from here.